src/main/java/com/blackducksoftware/integration/hub/service/model/ScannerSplitStream.java

```diff
@@ -220,36 +221,42 @@ public void close() throws IOException {
     }
 
     private void writeToConsole(final String line) {
-        if (StringUtils.containsIgnoreCase(line, DEBUG) || StringUtils.containsIgnoreCase(line, TRACE)) {
+        String trimmedLine = line.trim();
+        if (doesStartWith(trimmedLine, DEBUG) || doesStartWith(trimmedLine, TRACE)) {
             // We dont want to print Debug or Trace logs to the logger
             return;
         }
         final StringBuilder outputBuilder = new StringBuilder();
         outputBuilder.append(output);
-        if (StringUtils.containsIgnoreCase(line, EXCEPTION)) {
-            // looking for 'Exception in thread' type messages
-            outputBuilder.append(line);
+        if (doesStartWith(trimmedLine, ERROR)) {
```

Comment:
It looks like some effort was done to turn strings into reasonable variables (ERROR, WARN, EXCEPTION, FINISHED, etc) so I see no reason to not also create LINE_SEPARATOR to reduce the noise in this method. You can replace all 'System.getProperty("line.separator")' occurrences with the variable. Seems cleaner to me.